src/Client/app/(admin)/invite-candidates/page.tsx

```
@@ -18,255 +17,368 @@ import {
+  
       if (columnKey === 'action') {
         return (
           <div className="flex justify-center gap-4">
-            <button onClick={() => setIsEditModalOpen(true)}>
+            <button
+              onClick={() => {
+                setSelectedEmail(user)
+                setEditedEmail(user.email)
+                setEditingEmail(user.email)
+              }}
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)